# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../acl/install.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- acl-2.2.32/install-sh.vanilla	2008-08-22 08:47:12.000000000 +0200
+++ acl-2.2.32/install-sh	2008-08-22 08:47:23.000000000 +0200
@@ -122,7 +122,7 @@
 
 if $INSTALL
 then
-    CP=cp; LN=ln; MKDIR=mkdir; CHMOD=chmod; CHOWN=_chown
+    CP=install; LN=ln; MKDIR=mkdir; CHMOD=chmod; CHOWN=_chown
 else
     CP=true; LN=true; MKDIR=true; CHMOD=true; CHOWN=true
 fi
